tests/wp-unit/include/Core/Posts/Modules/DeletePostsByTaxonomyModuleTest.php

```diff
@@ -26,6 +26,9 @@ public function setUp() {
 		$this->module = new DeletePostsByTaxonomyModule();
 	}
 
+	/**
+	 * Test deleting posts from a single taxonomy term default post type.
+	 */
 	public function test_deleting_posts_from_built_in_taxonomy_terms() {
```

Comment:
L#48 `private` filter is not required here, as **we do not have an option in the UI** for users to set this.